src/main/java/complete_task/completeTaskInputBoundary.java

```diff
@@ -9,7 +10,8 @@
 public interface completeTaskInputBoundary {
```

Comment:
I don't think these attributes should be here. Interface attributes are always public static and final, so these will be initialized to null and 0

Reply:
Also, I believe the only reason this does not throw an exception is because of the Singleton pattern (since it auto-initializes the class), so this would abuse the purpose of the pattern in my opinion and constitute poor architecture.